docs/reference/cluster/stats.asciidoc

```diff
@@ -38,6 +38,278 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=flat-settings]
+
+`indices.shards.index.primaries.max`::
+(integer)
+Highest historic number of primary shards for the cluster.
+
+`indices.shards.index.primaries.avg`::
+(integer)
+Average number of primary shards for the cluster.
+
+`indices.shards.index.replication.min`::
+(integer)
+Lowest historic number of replica shards for the cluster.
+
+`indices.shards.index.replication.max`::
+(integer)
+Highest historic number of replica shards for the cluster.
+
+`indices.shards.index.replication.avg`::
+(integer)
+Average number of replica shards for the cluster.
+
+`indices.docs.count`::
+(integer)
+Number of non-deleted documents in the cluster.
++
+This number is based on documents in Lucene segments and may include documents
+from nested fields.
+
+`indices.docs.deleted`::
+(integer)
+Number of deleted documents in the cluster.
++
+This number is based on documents in Lucene segments. {es} reclaims the disk
+space of deleted Lucene documents when a segment is merged.
+
+`indices.store.size`::
+(<<byte-units, byte units>>)
+Size of the indices store.
+
+`indices.store.size_in_bytes`::
+(integer)
+Size, in bytes, of the store.
+
+`indices.fielddata.memory_size`::
+(<<byte-units, byte units>>)
+Memory used for fielddata cache.
+
+`indices.fielddata.memory_size_in_bytes`::
+(integer)
+Memory, in bytes, used for fielddata cache.
+
+`indices.fielddata.evictions`::
+(integer)
+Number of fielddata evictions.
+
+`indices.query_cache.memory_size`::
+(<<byte-units, byte units>>)
+Memory used for query cache.
+
+`indices.query_cache.memory_size_in_bytes`::
+(integer)
+Memory, in bytes, used for query cache.
+
+`indices.query_cache.total_count`::
+(integer)
+Total count of hits, misses, and cached queries in the query cache.
+
+`indices.query_cache.hit_count`::
+(integer)
+Number of query cache hits.
+
+`indices.query_cache.miss_count`::
+(integer)
+Number of query cache misses.
+
+`indices.query_cache.cache_size`::
+(integer)
+Size, in bytes, of the query cache.
+
+`indices.query_cache.cache_count`::
+(integer)
+Count of queries in the query cache.
+
+`indices.query_cache.evictions`::
+(integer)
+Number of query cache evictions.
+
+`indices.completion.size_in_bytes`::
+(<<byte-units, byte units>>)
+Memory used for completion.
+
+`indices.completion.size_in_bytes`::
+(integer)
+Memory, in bytes, used for completion.
+
+`indices.segments.count`::
+(integer)
+Number of segments.
+
+`indices.segments.memory`::
+(<<byte-units, byte units>>)
+Size of segments.
+
+`indices.segments.memory_in_bytes`::
+(integer)
+Size, in bytes, of segments.
+
+`indices.segments.terms_memory`::
+(<<byte-units, byte units>>)
+Size of terms in segments.
+
+`indices.segments.terms_memory_in_bytes`::
+(integer)
+Size, in bytes, of terms in segments.
+
+`indices.segments.stored_fields_memory`::
+(<<byte-units, byte units>>)
+Size of stored fields in segments.
+
+`indices.segments.stored_fields_memory_in_bytes`::
+(integer)
+Size, in bytes, of stored fields in segments.
```

Comment:
Total amount of memory used for stored fields across all shards etc. etc. (stored fields also kinda exist on disk)

Reply:
Fixed with 3e4b778913b.